src/conditionals/simple_programs_with_decision_points/medium/q1/InvestOpTest.java

```diff
@@ -0,0 +1,37 @@
+package conditionals.simple_programs_with_decision_points.medium.q1;
+
+/*
+Write a program that will ask the user for their investment and double it if it is above $5000
+ */
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.RandomInteger;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.RepeatedTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.Random;
+
+
+public class InvestOpTest extends BaseTest {
+    public Random r = new Random();
```

Comment:
Also, why the random? From what I see it isn't used, but I might be missing something

Reply:
Yeah, this one is still a draft, so it is not actually ready for review yet